go-dlv.el

```diff
@@ -131,6 +132,19 @@ and source-file directory for your debugger."
   (setq paragraph-start comint-prompt-regexp)
   (run-hooks 'go-dlv-mode-hook))
 
+;;;###autoload
+(defun dlv-current-test ()
```

Comment:
> How about having two functions instead: dlv-func-at-point and dlv-at-point? The only difference between them would be a breakpoint location, current function (will work with tests too) or current line.

With dlv-func-at-point, do you only mean the test functions, or functions in general? 

In any case, we can leave those plans out of this PR. If needed, it can be done later. We might want to consider renaming dlv-current-test to dlv-current-func, just in case someone wants to extend the functionality to include normal non-test functions (based on dlv debug instead of dlv test).